Gemfile

```diff
@@ -29,7 +29,11 @@ gem 'coffee-rails', '~> 4.1.0'
 # gem 'therubyracer', platforms: :ruby
 
 # use GDS assets, styles etc...
-gem 'govuk_frontend_toolkit', '~> 4.2.1'
+gem 'govuk_frontend_toolkit',
+  git: "https://github.com/alphagov/govuk_frontend_toolkit_gem.git", submodules: true
```

Comment:
I'm slightly concerned about using `master` versions from Git. I'll have a look if we should try to peg it to a version or what are the alternatives.